### PR TITLE
Mission Support System - suborg information for gsoc 2020

### PR DESCRIPTION
**Pull request by ReimarBauer**
Hi there,

happy new year!

We like to participate with our project in GSOC 2020. We will add in the next days further project ideas  
to our page.  Last year this was a big success for us.

Let us know if something is missing.

cheers
Reimar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -431,6 +431,35 @@ <h2>
...
+                <br />
+                MSS is a web service based client/server application to plan atmospheric research flights. <br />
+                <p>
+                The Mission Support System (MSS) is a software that is written by scientists in the field of atmospheric science. The purpose is to have a tool that simplifies the process for planning a scientific flight in which parameters of the atmosphere are measured. The research aircrafts typically carry a comprehensive scientific payload comprised of data aquisition instruments by different companies and research institutions. The measurement of relevant parameters, for example the chemical composition of trace gases, temperature or aerosol particle characteristics, are needed to improve the scientific understanding of the processes in the atmosphere. This is of significant importance for the understanding for example of climate change or the recovery of the ozone hole. </p>
+                <div class="pure-g">
+                  <div class="pure-g" style="text-align: center;">
+￼						<div class="pure-u-1 pure-u-md-1-4"><p><a href="https://bitbucket.org/wxmetvis/mss/wiki/GSOC2020/Website">Website</a></p></div>
```
Comment (terriko) [inline index.html]: This link seems to be broken.  What should it be?

**Comment by terriko** [comment]
Also, if you didn't get the new signup link for our volunteer management system, send a ping to the gsoc-admins to get it.  And if you did, please sign up there too -- we'll be using it for blog notifications and stuff.

File: index.html (hunk 1 above)
Comment (ReimarBauer) [reply index.html]: I redo this, not sure why the merge looks like this

**Comment by ReimarBauer** [comment]
I redo this completly.

**Pull request closed without merging**